compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -282,6 +282,14 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                             "you may want to use a bool value instead",
                             format!("{}", item_typo),
                         ))
+                    // FIXME(vicnenzopalazzo): make the check smarter,
+                    // and maybe expand with levenshtein distance checks
+                    } else if item_str.as_str() == "printf" {
+                        Some((
+                            item_span,
+                            "you may want to use the macro `print!(..)`",
```

Comment:
If you can change the wording here to be 
```suggestion
                            "you may have meant to use the `print` macro",
```

Reply:
Done!